### DISCUSSION TITLE
syntax error: invalid regular expression given adjacent forward-slash and open-paren

### DISCUSSION THREAD

**Discussion by phrohdoh** [athensresearch/athens#867] [Archive]
(*edit*: observed on Athens 1.0.0-beta.60)

to reproduce type out the following in a block: `/(f` (`f` is just an example, perhaps the user is going to reference "fred")

<img width="973" alt="Screen Shot 2021-03-22 at 9 16 04 AM" src="https://user-images.githubusercontent.com/4861023/112003812-3bcc6580-8aef-11eb-9613-3cb2a00d90a5.png">

as you can see from the error message, `/(f/` is what is being read as a regex (even though in Athens I have `/(|)`, read `|` as the cursor)

note that, for example, `meet w/ (` works as desired

some questions I've come up with:
- why is this happening? are we using `eval` (😱) somewhere?
- what can we do to _not_ interpret user-entered text as a JS regex?

**Comment by phrohdoh** [comment]
I've run into exactly this again (this time in my page's title) today.

**Reply by phrohdoh** [reply]
A page with a title like this can not be renamed nor can it be deleted.
The syntax error occurs and the page remains as it was prior.

**Comment by jsmorabito** [comment]
replicated in beta.64 on windows 10
